Vocable/Features/Settings/EditCategories/EditCategoryDetailViewController.swift

```diff
@@ -233,7 +233,9 @@ final class EditCategoryDetailViewController: VocableCollectionViewController, E
             return true
         case .showCategoryToggle:
             return (category.identifier != .userFavorites)
-        case .addPhrase, .removeCategory:
+        case .addPhrase:
+            return category.identifier == Category.Identifier.userFavorites ? true : category.isUserGenerated
```

Comment:
As discussed: ternary is probably unnecessary here

Reply:
```suggestion
            return (category.identifier == Category.Identifier.userFavorites) || category.isUserGenerated
```
If I'm reading that right?